Configurateur/front/lib/screens/register/register.dart

```diff
@@ -35,230 +39,260 @@ class RegisterScreenState extends State<RegisterScreen> {
+                                  ]);
+                                }
+                                await HttpService().request(
+                                    'http://$serverIp:3000/api/auth/register-confirmation',
+                                    header,
+                                    body);
+                                // ignore: use_build_context_synchronously
+                                context.go("/register-confirmation",
+                                    extra: mail);
+                              }
+                            },
+                            style: ElevatedButton.styleFrom(
+                              shape: RoundedRectangleBorder(
+                                borderRadius: BorderRadius.circular(30.0),
+                              ),
+                            ),
+                            child: Text(
+                              "S'inscrire",
+                              style: TextStyle(
+                                  color:
+                                      Provider.of<ThemeService>(context).isDark
+                                          ? darkTheme.primaryColor
+                                          : lightTheme.primaryColor,
+                                  fontSize: 18),
                             ),
                           ),
                         ),
-                      ),
-                      InkWell(
-                        key: const Key('login'),
-                        onTap: () {
-                          context.go("/login");
-                        },
-                        child: Padding(
-                          padding: const EdgeInsets.all(10.0),
-                          child: Row(
-                              mainAxisAlignment: MainAxisAlignment.center,
-                              children: const <Widget>[
-                                Text("Déja un compte ? "),
-                                Text(
-                                  'Connectez-vous.',
-                                  style: TextStyle(color: Colors.blue),
-                                ),
-                              ]),
+                        InkWell(
+                          key: const Key('login'),
+                          onTap: () {
+                            context.go("/login");
+                          },
+                          child: const Padding(
+                            padding: EdgeInsets.all(10.0),
+                            child: Row(
+                                mainAxisAlignment: MainAxisAlignment.center,
+                                children: <Widget>[
+                                  Text("Déja un compte ? "),
+                                  Text(
+                                    'Connectez-vous.',
+                                    style: TextStyle(color: Colors.blue),
+                                  ),
+                                ]),
```

Comment:
Maybe add , after the ] so that the file is more readable

Reply:
done